Makefile

```
@@ -0,0 +1,30 @@
+CONTAINER_CMD := $(or $(CONTAINER_CMD), $(shell command -v podman 2> /dev/null))
+ifndef CONTAINER_CMD
+CONTAINER_CMD := docker
+endif
+
+BUG_MASTER_IMAGE := $(or $(BUG_MASTER_IMAGE), "quay.io/edge-infrastructure/bug-master")
+BUG_MASTER_TAG := $(or $(BUG_MASTER_TAG),latest)
+
+build-image:
+	$(CONTAINER_CMD) build $(CONTAINER_BUILD_EXTRA_PARAMS) -t $(BUG_MASTER_IMAGE):$(BUG_MASTER_TAG) .
+
+install:
+	pip install -r requirements.txt
```

I think you need:
```suggestion
	pip install -r requirements.txt .
```

otherwise ``bug-master`` entrypoint won't be available as the package is not installed

---

Indeed